stream_alert/alert_processor/outputs.py

```diff
@@ -199,6 +196,245 @@ def dispatch(self, **kwargs):
         return self._log_status(success)
 
 @output
+class PagerDutyIncidentOutput(StreamOutputBase):
+    """PagerDutyIncidentOutput handles all alert dispatching for PagerDuty Incidents API v2"""
+    __service__ = 'pagerduty-incident'
+    INCIDENTS_ENDPOINT = 'incidents'
+    USERS_ENDPOINT = 'users'
+    POLICIES_ENDPOINT = 'escalation_policies'
+    SERVICES_ENDPOINT = 'services'
+
+    def __init__(self, *args, **kwargs):
+        StreamOutputBase.__init__(self, *args, **kwargs)
+        self.base_url = None
```

Comment:
Maybe make these instance properties protected? ie - `self._headers`, etc..  since they should never be accessed from outside an instance of this class.